azkaban-web-server/src/main/java/cloudflow/models/ExecutionBasicResponse.java

```diff
@@ -22,8 +22,8 @@ public ExecutionBasicResponse(ExecutableFlow executableFlow) {
     this.submitTime = executableFlow.getSubmitTime();
     this.concurrentOption = executableFlow.getExecutionOptions().getConcurrentOption();
     this.failureAction = executableFlow.getExecutionOptions().getFailureAction().toString();
-    this.isNotifyFailureFirst = executableFlow.getExecutionOptions().getNotifyOnFirstFailure();
-    this.isNotifyFailureLast = executableFlow.getExecutionOptions().getNotifyOnLastFailure();
+    this.notifyOnFirstFailure = executableFlow.getExecutionOptions().getNotifyOnFirstFailure();
+    this.notifyFailureOnExecutionComplete = executableFlow.getExecutionOptions().getNotifyOnLastFailure();
```

Comment:
Current names for these properties in Azkaban are unclear or not easy to understand. Let's agree on better names for the POC.